movie/src/hooks/useIsOnMobile.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import { useEffect } from "react";
+
+const getIsSamsung = () => window.innerWidth <= 1080;
+const getIsHumbergerIcon = () => window.innerWidth <=1080;
+export default function useIsOnSamsung() {
+    const [isSamsung, setIsSamsung] = useState(getIsSamsung());
+
+    useEffect(() => {
+        const onResize = () => {
+            setIsSamsung(getIsSamsung());
+        }
+        window.addEventListener("resize", onResize);
+    
+        return () => {
+            window.removeEventListener("resize", onResize);
+        }
+    }, []);
+    
+    return isSamsung;
+}
+
+export const useIsOnShowHumbergerIcon = () => {
```

Comment:
why are there 2 of the same hook in this file? what is the difference? since the only difference I see are the names.